### DISCUSSION TITLE
Error - wtsexporter : The term 'wtsexporter' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path was included, verify that the path is correct and try again.

### DISCUSSION THREAD

**Discussion by ghost** [KnugiHK/WhatsApp-Chat-Exporter#52] [General]
I am trying to run the program for a decryped iphone backup, yet after following the last step:

wtsexporter -i -b "C:\Users\[Username]\AppData\Roaming\Apple Computer\MobileSync\Backup\[device id]"

I get this message on powershell. 

wtsexporter : The term 'wtsexporter' is not recognized as the name of a cmdlet, function, script file, or operable
program. Check the spelling of the name, or if a path was included, verify that the path is correct and try again.

Is there any way to fix this?

**Comment by KnugiHK** [comment]
Does Python Script directory's path include in your environment? You can check the environment variables in Powershell with `$env:path`.

**Reply by hlvt** [reply]
Hi, I'm having the same issue. I added `"C:\Users\[MY USERNAME]\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311"` as a path in the advanced system settings (Windows) but it still doesn't work. I tried using `wtsexporter --help` and it didn't work either.

![image](https://github.com/KnugiHK/WhatsApp-Chat-Exporter/assets/139316868/d22973ea-5baa-4e19-96c4-f5fe79c9f181)

**Reply by KnugiHK** [reply]
You should also add `C:\Users\[MY USERNAME]\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\Scripts` to PATH.
